### PR TITLE
[DF] Misc minor improvements to RCsvDS

### PR DESCRIPTION
**Pull request by eguiraud**

### CODE REVIEW

File: tree/dataframe/src/RCsvDS.cxx [hunk 1]
```diff
@@ -465,7 +465,7 @@ std::vector<std::pair<ULong64_t, ULong64_t>> RCsvDS::GetEntryRanges()
 
    const auto chunkSize = nRecords / fNSlots;
    const auto remainder = 1U == fNSlots ? 0 : nRecords % fNSlots;
-   auto start = 0ULL == fEntryRangesRequested ? 0ULL : fProcessedLines;
```
Comment (etejedor) [inline tree/dataframe/src/RCsvDS.cxx:468]: What's the reason for this change?

Comment (eguiraud) [reply tree/dataframe/src/RCsvDS.cxx:468]: you might also ask what was the reason for the previous version...I couldn't find any, to me it seems that if `fEntryRangesRequested` is 0 then `fProcessedLines` must be 0 too

Comment (etejedor) [reply tree/dataframe/src/RCsvDS.cxx:468]: But the previous logic relied on the value of `fEntryRangesRequested` to assign `start` to either `0` or `fProcessLines`, and the new logic always assigns to `fProcessedLines`. You're saying this change can be safely made?

Comment (eguiraud) [reply tree/dataframe/src/RCsvDS.cxx:468]: yes, see my previous message, but please do confirm that's the case :)

**Review by etejedor — APPROVED** [review]
Thanks!
